src/components/modules/banners/digitalSubscriptions/DigitalSubscriptionsBanner.stories.tsx

```diff
@@ -28,10 +28,15 @@ const tracking: BannerTracking = {
 
 export const defaultStory = (): ReactElement => {
     const content: BannerContent = {
-        heading: text('heading', 'Enjoy ad-free reading and the best of our apps'),
+        heading: text('heading', 'Progressive journalism'),
+        secondaryHeading: text('secondHeading', 'Powered by you'),
+        highlightedText: text(
+            'highlightedText',
+            'Two apps, to discover at your pace, uninterrupted by advertising.',
+        ),
         messageText: text(
             'messageText',
-            'Support the Guardian with a Digital Subscription, enjoy our reporting without ads and get premium access to our Live app and The Daily.',
+            'The Guardian digital subscription gives you full access to the Guardian’s Live and Daily app for you to enjoy whenever and wherever you like.',
```

Comment:
Am I correct in thinking that the text here will be fetched from the banner tooling for this banner?

Reply:
Yes, hopefully we should have that up and running before the end of the week, but otherwise the text is provided as default props in the banner test file.